ci/cron/wednesday.yml

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
It looks like we need a new pipeline for this. At the moment it doesn’t seem to get triggered from anywhere.